<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>depositcalc</class>
 <widget class="QWidget" name="depositcalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:	#FFF5EE</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма вклада</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sum_dep">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #ffffff;
    border: 1px solid #A9A9A9;
    border-radius: 10px;
    padding: 5px;
    font-size: 14px;
    color: #333;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>руб.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Срок размещения</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="time_dep">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #ffffff;
    border: 1px solid #A9A9A9;
    border-radius: 10px;
    padding: 5px;
    font-size: 14px;
    color: #333;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="procent_dep">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #ffffff;
    border: 1px solid #A9A9A9;
    border-radius: 10px;
    padding: 5px;
    font-size: 14px;
    color: #333;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Налоговая ставка</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="taxes_dep">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #ffffff;
    border: 1px solid #A9A9A9;
    border-radius: 10px;
    padding: 5px;
    font-size: 14px;
    color: #333;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Периодичность выплат</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    background-color: white;
    border: 1px solid gray;
    padding: 5px;
    selection-background-color: lightblue;
}
QComboBox QAbstractItemView {
    border: 1px solid gray;
    selection-background-color: lightblue;
    background-color: white;
}</string>
   </property>
   <item>
    <property name="text">
     <string>Каждый день</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Каждую неделю</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в месяц</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="capitalization">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Капитализация процентов</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Пополнения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Снятия</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Начисленные проценты</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="acc_perc">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>410</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #ffffff;
    border: 1px solid #A9A9A9;
    border-radius: 10px;
    padding: 5px;
    font-size: 14px;
    color: #333;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма налога</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sum_tax">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>450</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #ffffff;
    border: 1px solid #A9A9A9;
    border-radius: 10px;
    padding: 5px;
    font-size: 14px;
    color: #333;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>500</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма на вкладе к концу срока</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sum_in_end">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>490</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #ffffff;
    border: 1px solid #A9A9A9;
    border-radius: 10px;
    padding: 5px;
    font-size: 14px;
    color: #333;
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="month_pop">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>240</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #ffffff;
    border: 1px solid #A9A9A9;
    border-radius: 10px;
    padding: 5px;
    font-size: 14px;
    color: #333;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Месяц</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sum_pop">
   <property name="geometry">
    <rect>
     <x>292</x>
     <y>240</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #ffffff;
    border: 1px solid #A9A9A9;
    border-radius: 10px;
    padding: 5px;
    font-size: 14px;
    color: #333;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Сумма</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="month_snyat">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>280</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #ffffff;
    border: 1px solid #A9A9A9;
    border-radius: 10px;
    padding: 5px;
    font-size: 14px;
    color: #333;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Месяц</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sum_snyat">
   <property name="geometry">
    <rect>
     <x>292</x>
     <y>280</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #ffffff;
    border: 1px solid #A9A9A9;
    border-radius: 10px;
    padding: 5px;
    font-size: 14px;
    color: #333;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Сумма</string>
   </property>
  </widget>
  <widget class="QPushButton" name="res_dep">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>350</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Стиль для кнопок с градиентом */
QPushButton {
    background:qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #FF69B4 stop:1 #FFA07A);
    border: 1px solid #FFA07A;
    border-radius: 20px;
    color: black;
    padding: 8px 16px;
    font-size: 13px;
    
}

/* Стиль при наведении */
QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4169e1, stop: 1 #BA55D3);
}

</string>
   </property>
   <property name="text">
    <string>Вычислить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>месяц.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
